charts/app-config/values-integration.yaml

```diff
@@ -143,16 +143,20 @@ govukApplications:
       hosts:
         - name: assets-origin.{{ .Values.k8sExternalDomainSuffix }}
           paths:
+            - path: /assets/
```

Comment:
I think this might take precedence over:

https://github.com/alphagov/govuk-helm-charts/blob/bef5709e4cfa23d37f2259a74cb8e12e638d49a2/charts/app-config/values-integration.yaml#L2892

Reply:
We use the same ALB to route to requests from the asset domain to Asset Manager, Whitehall and Static depending on path - need to make sure this rule isn't going capture requests currently going to Whitehall

Reply:
Ah snap hehe :) Let's find out who guessed right 😂

Reply:
Cool, thanks both.  I'll check the behaviour is correct after it's been rolled out.

Reply:
Just in case you end up having to dive into this: https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.5/guide/ingress/annotations/#group.order (in theory should be ok because lower number => its listener rules come first => should work like you want, but entirely possible I've missed something else)

Reply:
> Just in case you end up having to dive into this: https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.5/guide/ingress/annotations/#group.order (in theory should be ok because whitehall-frontend has the lower number => its listener rules come first => should work like you want, but entirely possible I've missed something else)

Thanks, that doc is a really interesting read.

This appears to have worked as planned, without (as far as I can see) any unintended consequences.